Sources/GithubAPI.swift

```diff
@@ -0,0 +1,97 @@
+/*
+ Copyright 2018 Google LLC
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ https://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+import Foundation
+import PerfectCURL
+import PerfectHTTP
+import PerfectLogger
+import PerfectCrypto
+
+public class GithubAPI {
+  static let githubBaseURL = "https://api.github.com"
+  static let retryCount = 0
+
+  class func addLabelsToIssue(url: String, labels: [String]) {
```

Comment:
Note: any modifications made via the GitHub API (POST/PATCH/DELETE/PUT) need to have a 1s delay if being done in batch.

Reply:
done